### ISSUE TITLE
Creation squelette de l'outil

### ISSUE THREAD

**Issue by cesar-richard** [simde-utc/gesasso2#24]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by cesar-richard** [comment]
Pour l'état de l'art on part apparemment sur Python et vous souhaitez continuer avec l'utilisation de Fabric. J'ai bien suivi @jumelloic @AudreyGl  ?

**Comment by AudreyGl** [comment]
Oui, à ce que j’ai compris Fabric est le bon outil pour exécuter des scripts Python en ligne de commande en local mais surtout sur des serveurs distants via SSH, ça permet d’avoir des scripts plus simples et plus propres.

**Comment by AudreyGl** [comment]
On utilise le package [smbldap-tools](https://www.mankier.com/package/smbldap-tools) ?

**Comment by cesar-richard** [comment]
> On utilise le package [smbldap-tools](https://www.mankier.com/package/smbldap-tools) ?

C'est ce qu'on fait jusque là, mais j'imagine que python doit bien embarquer une dependence qui fait le café non ? plutot que de s'appuyer sur la config de la machine hote

**Comment by AudreyGl** [comment]
Aaah oui ok, [ldaptor](https://pypi.org/project/ldaptor/) a l'air de correspondre à ça \o/

**Comment by cesar-richard** [comment]
@AudreyGl @jumelloic Ce qui serait bien pour une v1 ca serait d'avoir déjà un outil qui permette juste de prendre des params en entrée et de faire de la sortie sur la console (voire sur syslog tout de suite si vous êtes chaud ! Et si vous savez ce que c'est surtout)
Meme si ca comprend juste 1 action possible qui dise HELLO <Name> ! C'est déjà ca.

De mon côté ca me débloquerait pour mettre en place le process de déploiement / test en preprod et prod :)
